go.sh

```diff
@@ -33,23 +33,24 @@ NB_HELP_MSG="
 $NB_THIS_FILE [options]
 
 options:
-   -h|--help                   Print this usage information and exit
+   -h|--help                 Print this usage information and exit
```

Comment:
Can you please indent this two spaces to right so that it aligns with the rest of the help message?